src/main/kotlin/uk/gov/justice/digital/hmpps/nomisvisitsmappingservice/casenotes/CaseNotesMappingResource.kt

```diff
@@ -319,6 +319,36 @@ class CaseNotesMappingResource(private val mappingService: CaseNoteMappingServic
     dpsCaseNoteId: String,
   ): CaseNoteMappingDto = mappingService.getMappingByDpsId(dpsCaseNoteId)
 
+  @GetMapping("/dps-casenote-id/{dpsCaseNoteId}/all")
+  @Operation(
+    summary = "get multiple mappings",
+    description = "Retrieves mappings by DPS id. In case of past merges, there could be > 1 nomis id per dps id. Requires role NOMIS_CASENOTES",
```

Comment:
I didn't spot any tests for the new endpoint...

Reply:
whoah good point ! 🤔